repo_resource/test_check.py

```diff
@@ -78,6 +78,21 @@ def setUp(self):
                 'name': 'aosp_remove_yukawa_project.xml'
             }
         }
+        self.include_one_project_source = {
+            'source': {
+                'url': 'https://github.com/makohoek/demo-manifests.git',
+                'revision': 'main',
+                'name': 'aosp_include_one_project.xml'
+            }
+        }
+
```

Comment:
nit: drop blank line please (to do the same as the other members)

Reply:
yes I will change it next version